package.json

```diff
@@ -78,7 +78,8 @@
       "@examples/trpc-next-prisma-starter>prisma": "https://registry.npmjs.com/prisma/-/prisma-4.10.1.tgz?id=%40examples%2Ftrpc-next-prisma-starter",
       "@examples/next-websockets-starter>prisma": "https://registry.npmjs.com/prisma/-/prisma-4.10.1.tgz?id=%40examples%2Fnext-websockets-starter",
       "@examples/trpc-next-prisma-todomvc>prisma": "https://registry.npmjs.com/prisma/-/prisma-4.10.1.tgz?id=%40examples%2Ftrpc-next-prisma-todomvc",
-      "@examples/legacy-next-starter>prisma": "https://registry.npmjs.com/prisma/-/prisma-4.10.1.tgz?id=%40examples%2Flegacy-next-starter"
+      "@examples/legacy-next-starter>prisma": "https://registry.npmjs.com/prisma/-/prisma-4.10.1.tgz?id=%40examples%2Flegacy-next-starter",
+      "@types/testing-library__jest-dom": "link:./packages/tests/vendor/@types/testing-library__jest-dom"
```

Comment:
Do we need this as a package even? Can we not just import the file in setupTests and it'll be registered?

Or is it a peerDep for something else?

Reply:
It's an indirect transitive dependency that we have to override, explained it in detail here https://github.com/trpc/trpc/issues/3916